lib/middleware/routes-output/submission-data-with-labels.js

```diff
@@ -0,0 +1,28 @@
+const {getInstanceController} = require('../../controller/controller')
+const {formatProperties} = require('../../page/page')
+
+const submissionDataWithLabels = (title, heading, subHeading, destination, userData) => {
+  const answersComponent = {
+    _id: 'page.summary.answers',
+    _type: 'answers',
+    onlyShowCompletedAnswers: false
+  }
+
+  const pageInstance = {
+    _type: 'output.pdf',
+    title: title,
+    sectionHeading: subHeading,
+    heading,
+    columns: 'full',
+    isPdf: true,
+    skipRedact: destination === 'team',
+    components: [answersComponent]
+  }
+
+  const answersController = getInstanceController(answersComponent)
+  answersController.preUpdateContents(answersComponent, userData, pageInstance)
```

Comment:
Commenting for future us: this seems to be setting some kind of global state for either `pageInstance` or `userData`.